### PR TITLE
Update dexs to use `generic_spot_v2_compatible_trades` macro

### PR DESCRIPTION
**Pull request by tomfutago**
linked to #4759

### CODE REVIEW

**Comment by tomfutago** [comment]
Updated `wombat` dex so far and I think `woofi` is another candidate, but I have a similar note to the other generic spot macro update (#5108) - again I think from/to Amount/Token should be other way around (cc: @ARDev097):

https://github.com/woonetwork/woofi_swap_smart_contracts?tab=readme-ov-file#iwoorouter-interface :

> /// swap fromToken -> toToken
> /// fromAmount the amount of fromToken to swap
> /// realToAmount the amount of toToken to receive
> function swap(
>   address fromToken,
>   address toToken,
>   uint256 fromAmount,
>   uint256 minToAmount,
>   address payable to,
>   address rebateTo
> ) external payable returns (uint256 realToAmount);

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @tomfutago

File: seeds/_sector/dex/woofi_avalanche_c_base_trades_seed.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
-avalanche_c,woofi,1,2022-05-10,0x4ab2a9385bda8de7df444e6f342c12dd70ad6300fdf3a351d80f46e119a7811c,122,0x50b7545627a5162f82a992c33b87adc75187b218,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,14499471,400000076,126322419554
-avalanche_c,woofi,1,2022-07-28,0xa1bcc8d3ec39c8694c2be30f5489091fe1bfe8ffd570841e55e00bef6ee3359c,6,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,17904502,8699999999999999,197657
-avalanche_c,woofi,1,2022-09-07,0xf48e0d36d6a51f82edecb408b20c1aa275fddc71f18d4118e734e3116c19d359,8,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,19631494,49970000,2714758315778573433
+avalanche_c,woofi,1,2022-11-08,0xae581cf151eacf9877f628e722514674f3f13b19abcce4e82808a2350526134a,25,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,22106915,33765119660094401667,565826462
+avalanche_c,woofi,1,2022-05-10,0x4ab2a9385bda8de7df444e6f342c12dd70ad6300fdf3a351d80f46e119a7811c,122,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,0x50b7545627a5162f82a992c33b87adc75187b218,14499471,126322419554,400000076
```
[the remark this answers is not in the document]
Comment (tomfutago) [reply seeds/_sector/dex/woofi_avalanche_c_base_trades_seed.csv:3]: That's how it looks to me 🤷‍♂️

[the remark this answers is not in the document]
Comment (jeff-dude) [reply seeds/_sector/dex/woofi_avalanche_c_base_trades_seed.csv:3]: similar to beethoven, i'll leave this open for a bit to see if we can resolve this universally

**Comment by jeff-dude** [comment]
revisiting here. from what i can see, you apply all projects in this PR to generic spot v2 macro. within that macro:
- token bought = `toAmount`
- token sold = `fromAmount`

in [the old lineage](https://github.com/duneanalytics/spellbook/blob/main/models/woofi/avalanche_c/woofi_avalanche_c_trades.sql#L25-L26), woofi has it flipped to:
- token bought = `fromAmount`
- token sold = `toAmount`
and the seed matched this to run through CI

i believe you are correct here. the docs read "ToAmount the amount of toToken to receive" -- i interpret to receive as bought. looking at snowtrace on an example tx_hash makes that seem true too

let's proceed here as-is. for testing purposes moving forward, we'll likely need to update old lineage to match or adopt the test query to flip them for particular projects

**Review by jeff-dude — APPROVED** [review]
slick setup here in the generic macro 🔥

**Comment by tomfutago** [comment]
> revisiting here. from what i can see, you apply all projects in this PR to generic spot v2 macro. within that macro:
> 
> * token bought = `toAmount`
> * token sold = `fromAmount`
> 
> in [the old lineage](https://github.com/duneanalytics/spellbook/blob/main/models/woofi/avalanche_c/woofi_avalanche_c_trades.sql#L25-L26), woofi has it flipped to:
> 
> * token bought = `fromAmount`
> * token sold = `toAmount`
>   and the seed matched this to run through CI
> 
> i believe you are correct here. the docs read "ToAmount the amount of toToken to receive" -- i interpret to receive as bought. looking at snowtrace on an example tx_hash makes that seem true too
> 
> let's proceed here as-is. for testing purposes moving forward, we'll likely need to update old lineage to match or adopt the test query to flip them for particular projects

Cool. Old lineage updated now too.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @tomfutago

**Pull request merged**